src/GraphQL.Tests/Bugs/Bug2279.cs

```diff
@@ -15,10 +15,8 @@ public void NoDuplicateTypeNames()
             var e = Should.Throw<InvalidOperationException>(() => s.Initialize());
             var t1 = typeof(Bug2279GraphType<int>).FullName;
             var t2 = typeof(Bug2279GraphType<string>).FullName;
-            //nl will be determined at compile time dictated by how git is configured to handle line endings, not what environment the code was ran in
-            var nl = @"
```

Comment:
@Shane32 I didn't understand why it didn't work for you. As you can see, there was already another test with the same message. It worked before. I changed both tests and they both work after.

Reply:
I dunno. 🤷‍♂️   👍

Reply:
But you're still using `@` which of course works.  I tried using `Environment.NewLine` and had a problem.  I didn't know there was a `@$` syntax for literals. 👍

Reply:
OK